### ISSUE TITLE
Cannot find module './configs_dev'

### ISSUE THREAD

**Issue by sharipov-ru** [blan4/MusicPeer#1]
Could you add an example of configs_dev.js to the repo?

```
Error: Cannot find module './configs_dev'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/ruslansharipov/source/nodejs/MusicPeer/configs.js:10:20)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
```

**Comment by senior-sigan** [comment]
@sharipov-ru 
This is example config.

``` js
module.exports = {
  port: 9000,
  clientID: '******',
  clientSecret: '*******',
  host: 'http://localhost:9000',
  sessionSecret: 'secret'
};
```

**Comment by senior-sigan** [comment]
resolved in #2
